doc/classes/TileMap.xml

```diff
@@ -280,7 +280,7 @@
 			Position for tile origin. See [enum TileOrigin] for possible values.
 		</member>
 		<member name="cell_y_sort" type="bool" setter="set_y_sort_enabled" getter="is_y_sort_enabled" default="false">
-			If [code]true[/code], the TileMap's children will be drawn in order of their Y coordinate.
+			[i]Deprecated. [/i] If [code]true[/code], the TileMap's children will be drawn in order of their Y coordinate. This future is superseded by [member Node2D.y_sort_enabled].
```

Comment:
~~This future is~~ In the future this is

Reply:
UGH Why am I typo-city recently. Thanks, good catch!